test/integration/query-interface.test.js

```diff
@@ -123,20 +123,20 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           schema: 'schema'
         });
       }).then(function() {
-          return self.queryInterface.addIndex({
+        return self.queryInterface.addIndex({
+          schema: 'schema',
+          tableName: 'table'
+        }, ['name', 'isAdmin'], {
+          logging: log
+        }, 'schema_table').then(function() {
+          return self.queryInterface.showIndex({
             schema: 'schema',
             tableName: 'table'
-          }, ['name', 'isAdmin'], {
-            logging: log
-          }, 'schema_table').then(function() {
-            return self.queryInterface.showIndex({
-              schema: 'schema',
-              tableName: 'table'
-            }, {logging: log}).then(function(indexes) {
-              expect(indexes.length).to.eq(1);
-              count = 0;
-            });
+          }, {logging: log}).then(function(indexes) {
+            expect(indexes.length).to.eq(1);
+            count = 0;
           });
+        });
```

Comment:
best to keep style changes out of feature changes, easier to review and reason about.
You can then submit separate style changes pull requests, or atleast separate commits :)